flutter_local_notifications/example/lib/main.dart

```diff
@@ -706,10 +746,9 @@ class _HomePageState extends State<HomePage> {
                       },
                     ),
                   ],
-                  if (!kIsWeb &&
-                      (Platform.isIOS || Platform.isMacOS)) ...<Widget>[
+                  if (Platform.isIOS) ...<Widget>[
                     const Text(
-                      'iOS and macOS-specific examples',
+                      'iOS-specific examples',
```

Comment:
Was there a reason for this change? I recall there being some applicable to macOS. I assume you were trying to do this only for adding in `_buildConfigureInAppToggle()` where I've not seen how to bring up the context menu for macOS that this should only be for iOS

Reply:
Good catch, this was an accident, I was cleaning up my other edits where I inadvertently referenced Mac in the iOS directory files' comments and must have messed this up